XRTK-Core/Packages/com.xrtk.core/Runtime/Utilities/Editor/MixedRealityEditorSettings.cs

```diff
@@ -115,19 +114,14 @@ public static void CheckSettings()
                 message += "- Change the Scripting Backend to use IL2CPP\n";
             }
 
-            if (!PlayerSettings.virtualRealitySupported)
-            {
-                message += "- Enable XR Settings for your current platform\n";
-            }
-
             if (EditorUserBuildSettings.selectedBuildTargetGroup == BuildTargetGroup.WSA)
             {
                 message += "- Enable Shared Depth Buffer in the XR SDK Settings\n";
             }
 
             message += "\nWould you like to make these changes?\n\n";
 
-            if (!forceTextSerialization || !il2Cpp || !visibleMetaFiles || !PlayerSettings.virtualRealitySupported)
```

Comment:
Since this is gone, we might be able to get away with creating the scriptable object settings for each `XRLoader` so that people pulling the xrtk won't have to. I guess that depends on if we want to just let Unity handle most of the stuff for us or if we wanna go full native

Reply:
Will need to be careful though in case they're already loaded? Prob should be easy enough to check